test/puppetlabs/trapperkeeper/services/status/status_service_test.clj

```diff
@@ -174,6 +176,7 @@
       app
       [jetty9-service/jetty9-service
        webrouting-service/webrouting-service
+       scheduler-service/scheduler-service
        status-service]
       (merge status-service-config
         {:web-router-service {:puppetlabs.trapperkeeper.services.status.status-service/status-service "/alternate-status"}})
```

Comment:
Maybe, you'd have to set the interval low and then wait a small amount of time before checking I think, might be a little racy

Yeah it doesn't seem worth testing that it works without the config since that's covered implicitly and it seems pretty low risk